tests/test_gui.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+import time
+import sys
+from GUI import *
+
```

Comment:
can include a comment of the export command needed in order to run the tests (or add the path line to this file)